src/Data/Impact.elm

```diff
@@ -403,6 +405,42 @@ getAggregatedScoreData defs getter =
         []
 
 
+getAggregatedScoreOutOf100 : Definition -> Mass -> Impacts -> Int
+getAggregatedScoreOutOf100 { trigram } totalMass impacts =
+    let
+        ln =
+            logBase e
+    in
+    impacts
+        |> perKg totalMass
+        |> getImpact trigram
+        |> Unit.impactToFloat
+        |> (\value ->
+                -- See the documentation at https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire/impacts-consideres/score-100
+                (ln 2077 - ln value) / ln 2 * 20
+           )
```

Comment:
I'd really like to see at least a soingle unit test added, just for the sake of non-regression control over time